CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ From version 2.6.0, the sections in this file adhere to the [keep a changelog](h
 * [#2011](https://github.com/Shopify/shopify-cli/pull/2011): Adds support for the Spin rewrite
 
 ### Changed
+* [#2001](https://github.com/Shopify/shopify-cli/pull/2001): Improve Bug Issue Template
+* [#2004](https://github.com/Shopify/shopify-cli/pull/2004): Template improvements II
```

Comment:
I wouldn't include these changes as they are not much relevant for the CLI users, arent' they?

```suggestion
```